view/components/features/login-form.tsx

```diff
@@ -26,6 +26,35 @@ export interface LoginFormProps {
 export function LoginForm({ ...props }: LoginFormProps) {
   const { t } = useTranslation();
   const [showPassword, setShowPassword] = useState(false);
+  const [emailError,setEmailError] = useState('');
+  const [passwordError,setPasswordError] = useState('');
```

Comment:
can you add a screen recording of the same on the description of the PR ? it helps us to identify the issues quickly,